src/vector/__init__.py

```diff
@@ -85,6 +86,23 @@ def _import_awkward() -> None:
 else:
     from vector.backends.awkward import VectorAwkward
 
+try:
+    import sympy
+except ImportError:
+    sympy = None  # type: ignore[assignment]
+    if not typing.TYPE_CHECKING:
+        VectorSympy = None
+else:
+    from vector.backends.sympy import (
+        MomentumSympy2D,
+        MomentumSympy3D,
+        MomentumSympy4D,
+        VectorSympy,
```

Comment:
SymPy's preferred capitalization includes a capital "P", but we ignore NumPy's preferred capitalization, too. This is fine. :heavy_check_mark:

(`MomentumSymPy4D` would be too bumpy!)